lib/cocoapods/sandbox/headers_store.rb

```diff
@@ -24,21 +24,34 @@ def initialize(sandbox, relative_path)
         @sandbox       = sandbox
         @relative_path = relative_path
         @search_paths  = []
+        @search_paths_cache = {}
+        @search_paths_key = Struct.new(:platform_name, :target_name)
```

Comment:
this should be made a constant, no need to create a new type of struct for each instance

Reply:
ah yes will update.